build-image/Dockerfile

```diff
@@ -1,26 +1,27 @@
-FROM golang:1.13.3-stretch
+FROM golang:1.14.4-stretch
 RUN apt-get update && apt-get install -y curl python-requests python-yaml file jq unzip protobuf-compiler libprotobuf-dev && \
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-RUN curl -sL https://deb.nodesource.com/setup_6.x | sh -
+RUN curl -sL https://deb.nodesource.com/setup_13.x | sh -
```

Comment:
While installing Node 13.x while building image I'm also seeing some errors:
```
Step 3/23 : RUN curl -sL https://deb.nodesource.com/setup_13.x | sh -
 ---> Running in 068ea5b2bde2
sh: 129: [: X_13.x: unexpected operator
sh: 85: [[: not found
```

These errors look going away if you change `| sh` to `| bash`